desktop/test/render-visdiff.js

```diff
@@ -7,9 +7,14 @@ import {app, BrowserWindow, ipcMain} from 'electron'
 import {resolveRoot, resolveRootAsURL} from '../resolve-root'
 import dumbComponentMap from '../shared/more/dumb-component-map.desktop'
 
-const outputDir = resolveRoot('screenshots')
+if (process.argv.length !== 3) {
+  console.log('Usage: electron render-visdiff.js DESTINATION')
```

Comment:
That being said I'm 100% ok to punt on this kind of thing in our own tool
chain if it doesn't spread (but I'm sure the fix was easy)

On Thursday, May 19, 2016, Max Goodman <notifications@github.com> wrote:

> In desktop/test/render-visdiff.js
> <https://github.com/keybase/client/pull/2822#discussion_r63980558>:
>
> > @@ -7,9 +7,14 @@ import {app, BrowserWindow, ipcMain} from 'electron'
> >  import {resolveRoot, resolveRootAsURL} from '../resolve-root'
> >  import dumbComponentMap from '../shared/more/dumb-component-map.desktop'
> >
> > -const outputDir = resolveRoot('screenshots')
> > +if (process.argv.length !== 3) {
> > +  console.log('Usage: electron render-visdiff.js DESTINATION')
>
> Good point -- I was just being lazy 😇
>
> Updated!
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2822/files/4f91146242bbffa667d21fb83d2d78f2ebbe2650#r63980558>
>